jumpstart-week2/README.md

```diff
@@ -0,0 +1,222 @@
+# Frontend Basics and Introduction to git
+
+Terminal cont.
+Git basics
+  - idea of version control
+  - local vs remote repos
+  - commit, push, pull
+Front end basics
+- HTML
+    - elements
+    - tags
+    - attributes
+- Basic CSS
+- Javascript
+
+
+Basic plan
+1. Have everyone clone the git repository
+- Open terminal
+- Go to a folder
+- Run a git clone
+
+2. Tell everyone to ignore most of the boiler plate, get everyone to be able to run our base code
+- Introduce the idea of node
+- npm install
+- npm run start
+- Open localhost, explain the idea
+
+3. Use our example code to lecture about the basics of HTML
+- Bring up file for the homepage
+- Talk about format of HTML
+- Introduce tags, attributes
+- divs & spans
+
+4. Add some basic color and positioning CSS to make a basic layout
+- Have a stylesheet that is imported into homepage
+- Introduce className and id and css selector syntax
+- Have a CSS to define a layout already, have students use class names to insert elements into the layout
+
+5. Walk through building out a simple post component
+
+
+## What is HTML, CSS and Javascript?
+
+HTML provides the basic structure of sites. What is on the page?
+
+CSS is used to control presentation, formatting, and layout. How does the page look?
+
+JavaScript is used to control the behavior of different elements. What can the page do?
+
+HTML, CSS, and Javascript are ever evolving technologies that were created around the year 2000, practically ancient in web development terms. They all have their own systems for adding new features and aren't always consistent with each other or even within themselves. Because of this, there are often dozens of ways to do the same thing with HTML, CSS and Javascript. This can be confusing to beginners or even experienced developers, but it's important to not get discouraged and to always keep an open mind to alternative approaches even if you don't need them.
+
+To learn more, [this blog](https://blog.hubspot.com/marketing/web-design-html-css-javascript) gives a good overview.
+
```

Comment:
Also, I don't know if this helps, but the best description I've heard of HTML/CSS/JS is
>HTML is like the bones
>CSS is like the skin
>JS is like the brain

And you can take this analogy as far as you want.

Reply:
I actually really like the description we have here, I think it's concise and descriptive.

Reply:
ok!